workflow-examples/src/main/java/com/redhat/parodos/examples/continued/complex/OnboardingAssessmentTask.java

```diff
@@ -37,7 +37,7 @@
  */
 public class OnboardingAssessmentTask extends BaseAssessmentTask {
 
-	private static final String INPUT = "INPUT";
+	static final String INPUT = "INPUT";
```

Comment:
Any reason to change visibility for this constant?

Reply:
OK

Reply:
I wouldn't change this at all. If you change this on the code, there is noway to identify that it's a big change affecting the thing, I'll define in the test

Reply:
@eloycoto you're right, changed.